src/components/NewGame/NewGame.module.scss

```diff
@@ -0,0 +1,116 @@
+@import '../../styles/colors.scss';
```

Comment:
NBD, but I read yesterday that [`sass` is planning to slowly deprecate `import` in favor of `use`](https://sass-lang.com/documentation/at-rules/import). Not sure how much work it would be to change now, but probably worth taking a brief look at. Certainly NBD for this PR.

Reply:
I haven't heard anything about that yet. I'll add a card to look into it.